src/Temporalio/Workflows/WorkflowInfo.cs

```diff
@@ -12,6 +13,8 @@ namespace Temporalio.Workflows
     /// <param name="CronSchedule">Cron schedule if applicable.</param>
     /// <param name="ExecutionTimeout">Execution timeout for the workflow.</param>
     /// <param name="Headers">Headers from when the workflow was started.</param>
+    /// <param name="LastFailure">Failure if this workflow run is a continuation of a failure.</param>
+    /// <param name="LastResult">Successful result if this workflow is a continuation of a success.</param>
```

Comment:
"workflow run was continued-as-new from a previous run which ended in a..."

Is a bit more clear I think